Dockerfile

```diff
@@ -1,14 +1,11 @@
-FROM folioci/alpine-jre-openjdk11:latest
-
-ENV TRUST_ALL_CERTIFICATES false
-
-ENV VERTICLE_FILE mod-calendar-fat.jar
-
-# Set the location of the verticles
-ENV VERTICLE_HOME /usr/verticles
-
-# Copy your fat jar to the container
-COPY target/${VERTICLE_FILE} ${VERTICLE_HOME}/${VERTICLE_FILE}
-
-# Expose this port locally in the container.
-EXPOSE 8081
+FROM folioci/alpine-jre-openjdk17:latest
+
+# Copy your fat jar to the container
+ENV APP_FILE mod-calendar-fat.jar
+# - should be a single jar file
+ARG JAR_FILE=./target/*.jar
+# - copy
+COPY ${JAR_FILE} ${JAVA_APP_DIR}/${APP_FILE}
+
+# Expose this port locally in the container.
+EXPOSE 8080

```

Comment:
What was the motivation for changing the port that is exposed in the Docker container?

I think it's a good idea to document this in the readme and release notes because it might require operators to change their system configurations for running containers from this image.

Reply:
This was changed during some testing at my university; some antivirus software the university used to use ran on that port and was causing a conflict.  I'll restore it to 8081, there's no need for it to be different anymore